src/did-manager/cheqd-did-provider.ts

```diff
@@ -81,6 +81,13 @@ export class CheqdDIDProvider extends AbstractIdentifierProvider {
 		return identifier
 	}
 
+	async createIdentifierRaw(payload: string): Promise<any> {
```

Comment:
As discussed, let's move the payload to the `options` any as object and move back to just `createIdentifier`.
We can easily keep just one method and inject options in a file, as discussed as well.